heudiconv/utils.py

```diff
@@ -319,7 +319,7 @@ def load_heuristic(heuristic):
         path, fname = op.split(heuristic_file)
         try:
             old_syspath = sys.path[:]
-            sys.path.append(path)
+            sys.path.append(0, path)
```

Comment:
should have been `insert` not `append` -- let's see if I can accept my own suggestion:
```suggestion
            sys.path.insert(0, path)
```